components/usage/pkg/server/server.go

```diff
@@ -35,6 +36,8 @@ type Config struct {
 	StripeCredentialsFile string `json:"stripeCredentialsFile,omitempty"`
 
 	Server *baseserver.Configuration `json:"server,omitempty"`
+
+	apiv1.UsageConfig
```

Comment:
I'd name this explicitly. It also allows you to be explicit about the `json` field name which guards against issues when you rename the `apiv1.UsageConfig` in the future.